socorro/lib/util.py

```diff
@@ -2,49 +2,11 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-from __future__ import print_function
-
 import collections
 
-from itertools import islice
 import six
 
 
-def chunkify(iterable, n):
```

Comment:
`more_itertools` has `chunked` which does the same thing, so might as well switch to that one.